src/services/app-session.ts

```diff
@@ -20,6 +20,11 @@ export interface ShopifyConnectionData {
   jwtToken: string;
 }
 
+export interface DopplerAPIConnectionData {
```

Comment:
I think that the type `DopplerAPIConnectionData` is not necessary. 

Or, if it is, it should be a private type of DopplerApiClient.